packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue

```diff
@@ -100,8 +102,26 @@ function mouseoverPort(event) {
 	const totalOffsetY = el.offsetTop + el.offsetHeight / 2 + 1;
 	const portPosition = { x: nodePosition.x + totalOffsetX, y: nodePosition.y + totalOffsetY };
 	emit('port-mouseover', portPosition);
-	// }
 }
+
+function appendPort(port: { type: string; label: string; value: string; direction: string }) {
+	const { type, label, value, direction } = port;
+	if (direction === 'in') {
+		inputs.value.push({ id: inputs.value.length.toString(), type, label, value });
+	} else if (direction === 'out') {
+		outputs.value.push({ id: outputs.value.length.toString(), type, label, value });
+	}
+}
```

Comment:
I would say leave `value` as any for now

I am also tempted to say you can't update your own "input" as they are provided by the previous operation. Having dealt with graph/node/ports that deals with directionality (e.g. the previous "simulation-wiring-diagram") it can be confusing to wrap your head around once the graph's structure becomes more complicated.